packages/background/src/utils/constants/networks.ts

```diff
@@ -555,39 +555,39 @@ export const INITIAL_NETWORKS: Networks = {
         rpcUrls: [`https://zksync-testnet-node.blockwallet.io`],
         defaultRpcUrl: `https://zksync-testnet-node.blockwallet.io`,
         blockExplorerUrls: ['https://goerli.explorer.zksync.io'],
-        blockExplorerName: 'zkSync Explorer',
+        blockExplorerName: 'zkSync Testnet Explorer',
         actionsTimeIntervals: { ...SLOW_TESTNET_TIME_INTERVALS_DEFAULT_VALUES },
         tornadoIntervals: {
             depositConfirmations: DEFAULT_TORNADO_CONFIRMATION,
             derivationsForward: DERIVATIONS_FORWARD,
         },
         nativelySupported: true,
     },
-    SCROLL_L1_TESTNET: {
-        name: 'scroll_l1_testnet',
-        desc: 'Scroll L1 Testnet',
-        chainId: 534351,
-        networkVersion: '534351',
+    ZKSYNC_ERA_MAINNET: {
+        name: 'zksync_era_mainnet',
+        desc: 'zkSync Era Mainnet',
+        chainId: 324,
+        networkVersion: '324',
         nativeCurrency: {
             name: 'Ether',
-            symbol: 'TSETH',
+            symbol: 'ETH',
             decimals: 18,
         },
         iconUrls: [
-            'https://raw.githubusercontent.com/block-wallet/assets/master/blockchains/scroll/info/logo.png',
+            'https://raw.githubusercontent.com/block-wallet/assets/master/blockchains/zksync/info/logo.png',
         ],
         hasFixedGasCost: false,
         enable: true,
-        test: true,
-        order: 8,
+        test: false,
+        order: 7,
```

Comment:
is this 10? Last "mainnet" (RSK) has 9

Reply:
good catch, copy & paste abuse. Fixed order and moved code to match mainnets